src/components/activity-page/managed-interactive/managed-interactive.tsx

```diff
@@ -274,15 +274,23 @@ export const ManagedInteractive: React.ForwardRefExoticComponent<IProps> = forwa
     }
   }));
 
+  const setShowDeleteDataButton = () => {
```

Comment:
I see this has already been merged, but `setShowDeleteDataButton` implies that the function sets something when it actually just returns a value, so `getShowDeleteDataButton` or `computeShowDeleteDataButton` might be more appropriate.

Reply:
@kswenson Thanks for the suggestion. I'll keep my eye out for a chance to change that in the future. I vacillated between `setShowDeleteDataButton` and `getShowDeleteDataButton` but didn't think either was quite right. I went with `setShowDeleteDataButton` but I think `computeShowDeleteDataButton` would be better.

Reply:
Another option would be `shouldShowDeleteDataButton()`.

Reply:
Yeah, slightly but definitely even better.